src/compiler/checker.ts

```diff
@@ -6611,17 +6643,45 @@ namespace ts {
                     const targetName = getInternalSymbolName(target, verbatimTargetName);
                     includePrivateSymbol(target); // the target may be within the same scope - attempt to serialize it first
                     switch (node.kind) {
+                        case SyntaxKind.VariableDeclaration:
+                            // commonjs require: const x = require('y')
+                            if (isPropertyAccessExpression((node as VariableDeclaration).initializer!)) {
+                                // const x = require('y').z
+                                const initializer = (node as VariableDeclaration).initializer! as PropertyAccessExpression; // require('y').z
+                                const uniqueName = factory.createUniqueName((getExternalModuleRequireArgument(node) as StringLiteral).text); // _y
+                                const specifier = getSpecifierForModuleSymbol(target.parent || target, context); // 'y'

```

Comment:
Since this is `require('y').z`, `target` is `z` and `target.parent` is `require('y')`. I fall back to `target` because, um, it MIGHT be right? And the declaration emitter seems to try to emit things even if they're not correct? Probably this whole block should check that `target.parent` is defined before proceeding. @weswigham what do you think?